content/docs/tutorials/certificate-defaults/README.md

```diff
@@ -0,0 +1,609 @@
+This essentially means the value of one of these fields is either not required, or is defaulted somewhere else.
+That somewhere else could be in the cert-manager code base, or indeed by the issuer that creates and returns the X.509 certificate.
+Let's explore how we can manipulate these values to be something custom and make the `Certificate` users life easier.
+
+We will set up some `ClusterPolicy` resources and `Certificate` resources in this tutorial. 
+We will make reference to a `ClusterIssuer` in the `Certificate` spec that doesn't exist, but for this tutorial the `ClusterIssuer` is not required as we will not actually be requesting certificates.
+That means anyone can follow this tutorial even without their own domain.
+
+> ⚠️ To make it easy to get started we are using cluster scoped `ClusterPolicy` resources.
+> You can scope your defaults to the namespace level through the use of `Policy` resources in the future, but that will not be covered in this tutorial.
+>
+
+## 1 - Defaulting optional fields
+
+In this section we will set 3 fields for all `Certificate` resource automatically.
+All 3 fields here are not required fields, but might need to be set depending on the platform and issuer preferences.
+These fields are:
+
+- Set a default value of: `revisionHistoryLimit: 2`.
+- Set a default value of `Always` under `spec.privateKey.rotationPolicy`.
+- Set defaults for all the `spec.privateKey` fields.
+
+> ℹ️ Note how these rules are tackling the first two of our [3 uses cases](#use-cases) set out above.
+
+1. Let's first take a look at the `ClusterPolicy`:
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cpol-mutate-certificate-defaults.yaml
+    ```
+    🔗 <a href="cpol-mutate-certificate-defaults.yaml">`cpol-mutate-certificate-defaults.yaml`</a>
+
+1. Apply the policy to the cluster and check that it is ready:
+
+    ```shell
+    kubectl apply -f cpol-mutate-certificate-defaults.yaml
+    kubectl get cpol
+    ```
+    
+    When the `ClusterPolicy` is ready the output should look like this:
+
+    ```log
+    NAME                            BACKGROUND   VALIDATE ACTION   READY   AGE   MESSAGE
+    0-mutate-certificate-defaults   true         Audit             True    4s    Ready
+    ```
+
+1. Now inspect the `Certificate` for testing with the name "test-revision":
+
+    ```yaml file=../../../../public/docs/tutorials/certificate-defaults/cert-test-revision.yaml
+    ```
+    🔗 <a href="cert-test-revision.yaml">`cert-test-revision.yaml`</a>
+
+    You can see that we have set the most minimal configuration currently possible, specifying only a DNS name for the certificate, where to save it (`secretName`) and the issuer to use to request the certificate (`issuerRef`).
+
+1. Use the following command to dry-run apply the certificate and then `diff` it against the original resource, to see how the defaults from our `ClusterPolicy` are applied:
```

Comment:
Oh I see to italic those words :+1:  to highlight it is a dry-run

Reply:
I left the comma in to make the sentence easier to read in two parts.

Reply:
I also used single asterisk instead of underscore which has the same effect. All examples in [markdown docs](https://www.markdownguide.org/basic-syntax/#italic) use asterisks.